engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -169,18 +171,29 @@ Module translateModule(Tree module) {
             bindings = cons(c, bindings);
             expr = doc.getExpression();
           }
+          if (expr instanceof Private priv) {
+            if (priv.getBody() != null) {
+              throw new IllegalStateException("Private token with body is not yet supported: " + priv);
```

Comment:
Please don't throw exceptions - they hurt robustness of the parser - use `translateSyntaxError`

Reply:
Fixed in 8f58cac1fdd513361f08224bef86c6d96eb09be3